README.md

```diff
@@ -245,19 +193,15 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project please do well to follow me up on the shared social media platforms, you can also reach out to me if any need arises.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
+## 🙏 Acknowledgments <a name="acknowledgements">
```

Comment:
- [ OPTIONAL] It would be nice to include the link to the template used for easy access to the design in the acknowledgement section of the _**README.md**_ file.  You can use this [Link](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). 👍